buildpipeline/windows.groovy

```diff
@@ -78,6 +78,8 @@ simpleNode('Windows_NT','latest') {
                     }
                 } else if (params.TGroup == 'uap') {
                     targetHelixQueues = ['Windows.10.Amd64.ClientRS2.Open']
+                } else if (params.TGroup == 'netfx') {
+                    targetHelixQueues = ['Windows.10.Amd64.ClientRS3']
```

Comment:
.groovy scripts are for CI runs, they should never go to non-.Open queues (these require auth and will reject the run)